examples/kyb-app/src/components/organisms/DynamicUI/StateManager/components/ActionsHandler/helpers/is-event-rule.ts

```diff
@@ -0,0 +1,8 @@
+import { EventRule, IRule } from '@app/domains/collection-flow';
+
+export const isEventRule = (rule: unknown): rule is EventRule => {
+  if (typeof rule !== 'object') return false;
+  if (typeof (rule as IRule).value !== 'object') return false;
```

Comment:
Use a `isObject` util. By also checking if the `value` property exists the type assertion could be removed.